CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+## 6.0.1(YYYY-MM-DD)
+
+Anyone using encrypted Realms are strongly advised to upgrade to this version or higher.
+
+### Enhancements
+* None
+
+### Fixed
+* Race condition using encrypted Realms could lead to the Realm ending up corrupted when the file increased in size. This could manifest as a wide array of different error messages. Most commonly seen has been "Fatal signal 11 (SIGSEGV) from Java_io_realm_internal_UncheckedRow_nativeGetString", "XXX" and "YYY". ([#6152](https://github.com/realm/realm-java/issues/6152), since 5.0.0)
```

Comment:
```suggestion
* When using encrypted Realms a race condition could lead to the Realm ending up corrupted when the file increased in size. This could manifest as a wide array of different error messages. Most commonly seen has been "Fatal signal 11 (SIGSEGV) from Java_io_realm_internal_UncheckedRow_nativeGetString", "XXX" and "YYY". ([#6152](https://github.com/realm/realm-java/issues/6152), since 5.0.0)
```